impl/src/main/java/org/jboss/weld/bootstrap/events/ProcessSimpleInjectionTarget.java

```diff
@@ -45,6 +45,7 @@ public ProcessSimpleInjectionTarget(BeanManagerImpl beanManager, AnnotatedType<X
     public void setInjectionTarget(InjectionTarget<X> injectionTarget) {
         checkWithinObserverNotification();
         this.injectionTarget = injectionTarget;
+        BootstrapLogger.LOG.setInjectionTargetCalled(getReceiver().getClass(), injectionTarget.getClass());
```

Comment:
What's the difference between passing instance and instance.getClass? It will be bit verbose but yes we can log both.